compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -251,6 +251,7 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
       {
       self()->setUseSSEForSinglePrecision();
       self()->setUseSSEForDoublePrecision();
+      self()->setSupportsAutoSIMD();
```

Comment:
My suggestion to use "#if 0" over just commenting it out was based on the OMR coding standard document saying to use "#if 0" to disable code instead of comments. Removing it should be okay so I think I'll go with that.